src/encoded/static/components/form.js

```diff
@@ -890,6 +891,13 @@ export class Form extends React.Component {
         return this.state.isDirty && this.state.isValid && !this.state.editor_error && !this.communicating;
     }
 
+    cancel() {
+        const link = url.parse(this.context.location_href, true);
+
+        // the last '/' is a hack to reload the page. In app.js, fallbackNavigate() did not without it.
+        this.context.navigate(`${link.pathname}/`, { reload: true });
+    }
```

Comment:
Thought you had to bind `this` for `cancel()` but now see you’re calling it in an arrow function.

Reply:
Opps. You are correct. I will correct it. I have been dealing hooks for too long.